### DISCUSSION TITLE
Add help texts to fields

### DISCUSSION THREAD

**Discussion by sdevalk** [netwerk-digitaal-erfgoed/solid-crs#304] [Objects]
The object fields may not be understood by every user. For example: what is the meaning of 'Type' or 'Objectnaam'? Is it possible to add help texts to the fields?

![add_help_info_to_fields](https://user-images.githubusercontent.com/606948/122071396-ff12c080-cdf6-11eb-8c3e-2b0c4724dbd1.png)

**Comment by lem-onade** [comment]
Our form element components support descriptions under the input fields. 
![image](https://user-images.githubusercontent.com/32491056/122072309-be677700-cdf7-11eb-9859-d9ab0bef4f2a.png)

Could you provide the text values for these descriptions?

**Comment by lem-onade** [comment]
I'll convert this issue to a discussion. A relevant issue already exists: https://github.com/netwerk-digitaal-erfgoed/solid-crs/issues/262

**Comment by ddeboer** [comment]
https://docs.google.com/spreadsheets/d/1l7BOQHgX_n1pLXV1G4R5UjSJPnyuSIaVwg_bPZtvHMc
